lib/pliny/sidekiq/middleware/server/log.rb

```diff
@@ -1,8 +1,7 @@
 module Pliny::Sidekiq::Middleware
   module Server
     class Log
-      def initialize(metric_prefix: nil)
-        @metric_prefix = metric_prefix
+      def initialize(opts={})
```

Comment:
Is this kept around for backward compatibility? I think this change will require a big version bump anyway so maybe just remove the initializer at the same time?